molecule/default/molecule.yml

```diff
@@ -23,15 +23,27 @@ platforms:
       - /run/lock
     volumes:
       - '/sys/fs/cgroup:/sys/fs/cgroup:ro'
+      - '/home/david/Desktop/ch:/var/lib/clickhouse/'
```

Comment:
I think this is heavily dependant on your personal environment :P

Reply:
done!